### PR TITLE
Feature/fix reports

### PR DESCRIPTION
**Pull request by boba-and-beer**

### CODE REVIEW

File: tests/unit/test_utils/test_imports.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ def test_datasets():
 
 
 def test_core():
-    from relevanceai.operations.cluster.ops import ClusterOps
+    from relevanceai.operations.cluster.cluster import ClusterOps
     from relevanceai.operations.dr.dim_reduction import DimReduction
```
Comment (jtwinrelevanceai) [inline tests/unit/test_utils/test_imports.py:24]: `from relevanceai.operations.dr.dr import DimReduction`

File: relevanceai/operations/dr/dim_reduction_ops.py [hunk 2]
```diff
@@ -40,20 +43,33 @@ def __init__(
         self.dr_field = dr_field
         self.verbose = verbose
         self.n_components = n_components
+        self.dataset_id = dataset_id
+        self.vector_fields = vector_fields
+        self.alias = alias
 
         super().__init__(credentials)
 
     def fit(
```
Comment (jtwinrelevanceai) [inline relevanceai/operations/dr/dim_reduction_ops.py:52]: `def operate`

File: relevanceai/operations/dr/dim_reduction_ops.py [hunk 3]
```diff
@@ -63,16 +79,25 @@ def fit(
...
-            vector_field=vector_fields[0],
+            vector_field=vector_fields[0],  # type: ignore
             documents=documents,
-            alias=alias,
+            alias=alias,  # type: ignore
             dims=self.n_components,
         )
 
-        return self.update_documents(dataset_id, dr_documents)
+        return self.update_documents(dataset_id=dataset_id, documents=dr_documents)  # type: ignore
```
Comment (jtwinrelevanceai) [inline relevanceai/operations/dr/dim_reduction_ops.py:103]: `_update_documents`

File: relevanceai/operations/operations.py [hunk 4]
```diff
@@ -49,21 +47,36 @@ def auto_cluster(
...
         )
+        return ops(dataset_id=self.dataset_id, vector_fields=vector_fields)
 
     def dr(
         self,
-        model,
-        n_components,
-        alias,
-        vector_fields,
+        model: Any,
```
Comment (jtwinrelevanceai) [inline relevanceai/operations/operations.py:58]: `model: Union[str, Any]` Users should be able to pass a string

**Review by jtwinrelevanceai — CHANGES_REQUESTED** [review]

File: relevanceai/operations/operations.py (hunk 4 above)
Comment (boba-and-beer) [reply relevanceai/operations/operations.py:58]: @jtwinrelevanceai Any includes `str`

**Review by jtwinrelevanceai — APPROVED** [review]

**Pull request merged**